app/ui/components/header.tsx

```diff
@@ -1,16 +1,15 @@
-import { Box, Button, Typography } from '@material-ui/core';
 import { LinkComponent } from './navigation/LinkComponent';
 
 export default function Header() {
   return (
-    <Box display="flex" bgcolor="grey.200" p={2} alignItems="center">
-      <Typography>Prestagile</Typography>
-      <Box>
+    <div>
+      <h1>Prestagile</h1>
+      <div>
         <LinkComponent
           target="/test"
-          child={<Button color="primary">Test</Button>}
+          child={<button color="primary">Test</button>}
```

Comment:
debris ?

Reply:
Same as Footer i'm going to put back the button component but the chakra ui insted of material ui

Reply:
Ok for me